acceptance/tests/tasks/task_download.rb

```diff
@@ -0,0 +1,119 @@
+require 'pxp-agent/config_helper.rb'
+require 'puppet/acceptance/environment_utils.rb'
+
+test_name 'task download' do
+
+  extend Puppet::Acceptance::EnvironmentUtils
+
+  PUPPETSERVER_CONFIG_FILE = windows?(master) ?
+    '/cygdrive/c/ProgramData/PuppetLabs/puppetserver/etc/conf.d/webserver.conf'
+  : '/etc/puppetlabs/puppetserver/conf.d/webserver.conf'
+
+  step 'Ensure each agent host has pxp-agent running and associated' do
+    agents.each do |agent|
+      on agent, puppet('resource service pxp-agent ensure=stopped')
+      create_remote_file(agent, pxp_agent_config_file(agent), pxp_config_json_using_puppet_certs(master, agent).to_s)
+      on agent, puppet('resource service pxp-agent ensure=running')
+
+      assert(is_associated?(master, "pcp://#{agent}/agent"),
+             "Agent #{agent} with PCP identity pcp://#{agent}/agent should be associated with pcp-broker")
+    end
+  end
+
+  step 'Create the downloadable tasks on the master' do
+    # Make the static content path
+    env_name = File.basename(__FILE__, '.*')
+    @static_content_path = File.join(environmentpath, mk_tmp_environment(env_name))
+
+    # Create the tasks
+    nix_task_body = "#!/bin/sh\n#Comment to change sha\necho $PT_message"
+    win_task_body = "REM Comment to change sha\n@echo %PT_message%"
```

Comment:
Use `@REM` to avoid printing that line when the script is run. Got a failure
```
  Test Case tests/tasks/task_download.rb reported: #<Minitest::Assertion: Output did not contain 'hello'.
--- expected
+++ actual
@@ -1 +1,2 @@
-"hello"
+"C:\\Program Files\\Puppet Labs\\Puppet\\pxp-agent\\bin>REM Comment to change sha
+hello"
>
    Test line: tests/tasks/task_download.rb:65:in `block (5 levels) in run_test'
```

Reply:
Thanks, I updated it and am running the tests locally